lighthouse-core/audits/audit.js

```diff
@@ -94,6 +101,42 @@ class Audit {
     };
   }
 
+  /**
+   * @param {!Audit} audit
+   * @param {!AuditResult} result
+   * @return {{score: number, scoreDisplayMode: string, informative: boolean, rawValue: boolean}}
+   */
+  static _normalizeAuditScore(audit, result) {
+    let score = typeof result.score === 'undefined' ? result.rawValue : result.score;
+    let informative;
+    let rawValue;
+
+    if (typeof score === 'boolean' || score === null) {
```

Comment:
this is kind of confusing since `result.score` comes in as `number|undefined`(even though on inspection the boolean comes from `rawValue`). Since `rawValue` has to be defined and has to be a number or boolean, how do you feel about
```js
let score = result.score === undefined ? Number(result.rawValue) : result.score;
```
or some variant on that